lib/renderer/webview/webview-attributes.js

```diff
@@ -1,8 +1,8 @@
 'use strict'
 
-const WebViewImpl = require('./web-view')
+const WebViewImpl = require('./webview')
 const guestViewInternal = require('./guest-view-internal')
-const webViewConstants = require('./web-view-constants')
+const webViewConstants = require('./webview-constants')
```

Comment:
After thinking about this more and reviewing the diff again, I'm not so sure about these renames, the constructor for this object is `new WebView`, not `new Webview` so it feels right to have it dashed in the file name(s), similar to `browser-window.js`.